nf_core/components/install.py

```diff
@@ -53,6 +55,14 @@ def install(self, component, silent=False):
             # Check modules directory structure
             self.check_modules_structure()
 
+        if isinstance(component, dict):
+            if component["git_remote"] is not None:
+                remote_url = component["git_remote"]
+                self.modules_repo = ModulesRepo(remote_url, self.branch)
+            else:
+                self.modules_repo = ModulesRepo(self.current_remote, self.branch)
```

Comment:
I think it is. If the user tries to download a subworkflow which uses modules from different organizations and a remote for a module is not specified, it tries to download from the last defined remote. Example:

- User downloads subworkflow get_genome_annotation from jvfe, which has 2 modules, one from jvfe and one from nf-core, defined like:
```yml
components:
  - prokka:
      git_remote: https://github.com/nf-core/modules.git
      branch: master
  - wget
```
- It downloads prokka from nf-core, and modulesrepo gets set to nf-core
- It would then try to download wget from nf-core as well.

But I believe it's more intuitive that if a remote is not set, it should try to download the module from the same remote the subworkflow is located in, hence this line.